doc/userguide/unix-socket.rst

```diff
@@ -14,7 +14,7 @@ The unix socket is always enabled by default.
 
 The creation of the socket is managed by setting enabled to 'yes' or 'auto'
 under unix-command in Suricata YAML configuration file: ::
-  
+
```

Comment:
These white space changes do not seem relevant to the commit

Reply:
They aren't but i decided not to do a separate commit for simple white-space removal.